docs/user/spark/spark_jts.rst

```diff
@@ -0,0 +1,136 @@
+Spark JTS
+---------
+
+The Spark JTS module provides a set of User Defined Functions (UDFs) and User
+Defined Types (UDTs) that enable executing SQL queries in spark that perform
+geospatial operations on geospatial data types.
+
+GeoMesa SparkSQL support builds upon the ``DataSet``/``DataFrame`` API present
+in the Spark SQL module to provide geospatial capabilities. This includes
+custom geospatial data types and functions, the ability to create a ``DataFrame``
+from a GeoTools ``DataStore``, and optimizations to improve SQL query performance.
+
+This functionality is located in the ``geomesa-spark-jts`` module:
+
+.. code-block:: xml
+
+    <dependency>
+      <groupId>org.locationtech.geomesa</groupId>
+      <artifactId>geomesa-spark-jts_2.11</artifactId>
+      // version, etc.
```

Comment:
we have some filtering on this in the build if you want to include the current version number

Reply:
i believe the variable replacement doesn't work on code blocks :/
maybe we should revisit and use maven filtering instead, right now we're using sphinx variables. (not for this pr)